nilearn/signal.py

```diff
@@ -26,20 +26,24 @@ def _standardize(signals, detrend=False, normalize=True):
 
     Parameters
     ==========
-    signals: numpy.ndarray
+    signals : numpy.ndarray
```

Comment:
> Do we have to put space before colons now?

Yes. It's always been like this in the numpy doc standard. But we keep
forgetting.